### ISSUE TITLE
Contributor's Meetup 2022-03-22

### ISSUE THREAD

**Issue by jalondonot** [openedx/community-wg#59]
## Date, time, and location

Tuesday March 22nd at 15:00 UTC ([Timezone converter](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Open+edX+Contributors+Meetup&iso=20220222T15&p1=1440&ah=1))

- [ical](https://calendar.google.com/calendar/ical/c_mvh2oito7s9n89g5nnkk49psks%40group.calendar.google.com/public/basic.ics)
- [gcal](https://calendar.google.com/calendar/embed?src=c_mvh2oito7s9n89g5nnkk49psks%40group.calendar.google.com)

## Agenda

Anyone can add additional agenda items by replying in the comments below, or on the [discourse forum thread](https://discuss.openedx.org/t/open-edx-contributors-meetup/1450).

### 1. Intro

- Meet & greet newcomers if any
- Recap of the items in this agenda: anything to add or remove?

### 2. Working group updates

  - Marketing (@ehuthmacher)
  - Release (@BbrSofiane, @nedbat, @regisb)
  - [Frontend](https://discuss.openedx.org/t/announcing-the-frontend-working-group/5769) (@arbrandes)
  - https://github.com/openedx/community-wg/issues/13 (@sarina)
  - [Data](https://github.com/orgs/openedx/projects/5) (@blarghmatey, @e0d)
  - [Deprecation](https://discuss.openedx.org/t/announcing-the-deprecation-working-group/6173) (@dianakhuang)
  - [Product](https://docs.google.com/document/d/1u-2ScDQlR1riE5TtRv-6xN28fKIKv9rATOzr9rjwZ1w/edit) (@jmakowski1123)

### 3. Events

- Updates about Lisbon event

### 4. New or ongoing projects

**Higher Priority**
- Open edX Roadmap (@jmakowski1123, @angonz)
- [Upgrades](https://docs.google.com/spreadsheets/d/11DheEtMDGrbA9hsUvZ2SEd4Cc8CaC4mAfoV8SVaLBGI/edit#gid=1850889545)
 https://github.com/openedx/frontend-wg/issues/2
- Django Backend APIs: [Django Signal Events](https://discuss.openedx.org/t/configuration-for-the-hooks-extension-framework/4527), [Django Filters](https://discuss.openedx.org/t/hooks-framework-filters/5753) (@felipemontoya)

**Lower Priority**

### 5. Improvements and blockers - Retrospective

* Discuss & figure out how to unblock items from the sprint retrospective: https://discuss.openedx.org/t/core-contributor-sprint-retro-march-4-18th/6876 (@antoviaque) 

## Forum thread
https://discuss.openedx.org/t/open-edx-contributors-meetup/1450

## To join
Subscribe to the [calendar](https://calendar.google.com/calendar/ical/c_mvh2oito7s9n89g5nnkk49psks%40group.calendar.google.com/public/basic.ics), find the meeting, and click on the join URL!

## Video recording & chat log
[ConMeetup-22-03-22](https://youtu.be/1hsc75Kjf0M)
[Chat log](https://drive.google.com/file/d/1IeI7NC-UUaKVHyflGQrO2B3Jg23RxTl9/view?usp=sharing)

**Comment by antoviaque** [comment]
@jalondonot @arbrandes I have added an item to the agenda, the 5th one: "Improvements and blockers - Retrospective", which is for discussing https://discuss.openedx.org/t/core-contributor-sprint-retro-march-4-18th/6876
